components/kyma-environment-broker/internal/storage/tests_utils.go

```diff
@@ -417,3 +324,105 @@ func clearDBQuery() string {
 		postsql.RuntimeStateTableName,
 	)
 }
+
+func createDbContainer(log func(format string, args ...interface{}), hostname string) (func(), Config, error) {
+
+	cli, err := client.NewClientWithOpts(client.FromEnv)
+	if err != nil {
+		panic(err)
```

Comment:
nit: here the function too appears to be able to forward the error, consider avoiding panicking